inference-engine/src/mkldnn_plugin/nodes/mkldnn_split_node.cpp

```diff
@@ -80,6 +87,15 @@ MKLDNNSplitNode::MKLDNNSplitNode(const std::shared_ptr<ngraph::Node>& op, const
     if (!isSupportedOperation(op, errorMessage)) {
         IE_THROW(NotImplemented) << errorMessage;
     }
+
+    if (ngraph::as_type_ptr<const ngraph::op::v1::Split>(op)) {
+        algorithm = SplitDefault;
```

Comment:
If this algorithm is not used anyhow I would suggest to remove it from enum.

Reply:
removed